requirements.txt

```diff
@@ -1,4 +1,4 @@
-slumber==0.7.0
+slumber==git+https://github.com/edx/slumber.git@a02e8ba381965e1de710b40af69590437adfb809#egg=slumber
```

Comment:
Do we need to provide a version (e.g. `egg=slumber==0.7.0-edx`) here @feanil? If so, we will need to update `setup.py` and actually create a release on the edx/slumber repo.